### PR TITLE
Fix spellcheck on Firefox

### PR DESCRIPTION
**Pull request by max-winderbaum**

### CODE REVIEW

File: src/component/contents/DraftEditorLeaf.react.js [hunk 1]
```diff
@@ -98,6 +98,10 @@ class DraftEditorLeaf extends React.Component {
       targetNode = child;
     } else if (child.tagName === 'BR') {
       targetNode = node;
+    } else if (child.firstChild && child.firstChild.className === 'gecko-patch-comment') {
+      // We've added an extra span (see DraftEditorTextNode.react.js) to prevent Firefox spellcheck from destroying
+      // our content. Skip past the span and the react text comment to get to the real text node.
+      targetNode = child.firstChild.nextSibling.nextSibling;
```
Comment (colinjeanne) [inline src/component/contents/DraftEditorLeaf.react.js:104]: Does this ensure the fix works in Chrome?

**Comment by max-winderbaum** [comment]
@colinjeanne Yes this fix now works both in Chrome and FF

File: src/component/contents/DraftEditorLeaf.react.js [hunk 2]
```diff
@@ -99,7 +99,9 @@ class DraftEditorLeaf extends React.Component {
     } else if (child.tagName === 'BR') {
       targetNode = node;
     } else {
-      targetNode = child.firstChild;
+      // We've added an extra span (see DraftEditorTextNode.react.js) to prevent Firefox spellcheck from destroying
+      // our content. Skip past the span and the react text comment to get to the real text node.
+      targetNode = child.firstChild.nextSibling.nextSibling;
```
Comment (intentionally-left-nil) [inline src/component/contents/DraftEditorLeaf.react.js:104]: It seems pretty scary to guarantee that this DOM structure will always exist. Although I guess there's no great answer here.

Comment (intentionally-left-nil) [reply src/component/contents/DraftEditorLeaf.react.js:104]: I guess I would rather crash and then get an issue in sentry.

**Review by intentionally-left-nil — APPROVED** [review]
LGTM. Thanks!

**Review by colinjeanne — APPROVED** [review]
:ship:

**Comment by colinjeanne** [comment]
Are we going to make a corresponding issue and PR into mainline Draft?

**Comment by max-winderbaum** [comment]
Yep - I'll hook it up

**Pull request merged**